### PR TITLE
Added supported platforms test matrix

### PR DESCRIPTION
**Pull request by kf6gpe**
Changes proposed in this pull request:
* Updates test matrix to match https://flutter.googlesource.com/recipes/+/refs/heads/master/recipes/firebaselab/firebaselab.py#40.

### CODE REVIEW

File: src/docs/resources/faq.md [hunk 1]
```diff
@@ -998,4 +999,4 @@ follow Apple's [guidelines][] for App Store submission.
 [web instructions]: /docs/get-started/web
 [`Widget`]: {{site.api}}/flutter/widgets/Widget-class.html
 [widgets]: /docs/development/ui/widgets
-[supported platforms]: /docs/development/tools/sdk/release-notes/supported-platforms
\ No newline at end of file
+[supported platfomrs]: /docs/development/tools/sdk/release-notes/supported-platforms].
```
Comment (sfshaza2) [inline src/docs/resources/faq.md]: OK....... You introduced a typo here. Also, delete the square bracket at the end.

File: src/docs/resources/faq.md [hunk 2]
```diff
@@ -400,9 +400,10 @@ on the edge.
 * We support developing Flutter apps with Android and iOS devices,
   as well as with Android emulators and the iOS simulator.
 
-* We test on a variety of low-end to high-end phones and tablets.
-  For a detailed list of the platforms on which we test, see 
-  our list of [supported platforms].
+* We test on a variety of low-end to high-end phones and tablets,
+  For a detailed list of platform supports by version and
+  an explanation of our support policy, see 
+  [supported platforms].
```
Comment (sfshaza2) [inline src/docs/resources/faq.md]: Please add empty square brackets after the bracketed text.

File: src/docs/resources/faq.md (hunk 1 above)
Comment (sfshaza2) [inline src/docs/resources/faq.md]: You introduced a typo. Also remove the square bracket at the end of the line.

**Review by sfshaza2 — CHANGES_REQUESTED** [review]
A couple minor fixes.

File: src/docs/development/tools/sdk/release-notes/supported-platforms.md [hunk 3]
```diff
@@ -23,13 +23,15 @@ but that the development team does not directly test or support.
 
 |Platform|Version              |
 |-------|----------------------|
+|Android|Windows SDK 30        |
```
Comment (dnfield) [inline src/docs/development/tools/sdk/release-notes/supported-platforms.md:26]:
```suggestion
|Android|Android SDK 30        |
```

File: src/docs/development/tools/sdk/release-notes/supported-platforms.md [hunk 4]
```diff
@@ -23,13 +23,15 @@ but that the development team does not directly test or support.
...
 |Android|Android SDK 29        |
 |Android|Android SDK 28        |
 |Android|Android SDK 27        |
 |Android|Android SDK 26        |
 |Android|Android SDK 25        |
 |Android|Android SDK 24        |
 |Android|Android SDK 23        |
+|Android|Android SDK 22        |
 |Android|Android SDK 21        |
 |Android|Android SDK 19        |
```
Comment (dnfield) [inline src/docs/development/tools/sdk/release-notes/supported-platforms.md:36]: We may want to call out here or below that SDK 20 is covered by testing 19 for Flutter's purposes.

File: src/docs/resources/faq.md (hunk 1 above)
Comment (kf6gpe) [reply src/docs/resources/faq.md]: Spooky. That was in an older version that I'd tested and fixed. Wonder what I fat-fingered during the merge! Fixing.

File: src/docs/development/tools/sdk/release-notes/supported-platforms.md (hunk 3 above)
Comment (kf6gpe) [reply src/docs/development/tools/sdk/release-notes/supported-platforms.md:26]: Good catch. Fixing.

**Comment by kf6gpe** [comment]
Thanks for the feedback. Dunno what happened with the errors in the link last time 'round. This should look better.

File: src/docs/resources/faq.md [hunk 5]
```diff
@@ -999,4 +998,4 @@ follow Apple's [guidelines][] for App Store submission.
 [web instructions]: /docs/get-started/web
 [`Widget`]: {{site.api}}/flutter/widgets/Widget-class.html
 [widgets]: /docs/development/ui/widgets
-[supported platfomrs]: /docs/development/tools/sdk/release-notes/supported-platforms].
+[supported platforms]: /docs/development/tools/sdk/release-notes/supported-platforms].
```
Comment (sfshaza2) [inline src/docs/resources/faq.md:1001]: Remove the unmatched square bracket at the end of the line.

**Review by dnfield — APPROVED** [review]
LGTM

File: src/docs/development/tools/sdk/release-notes/supported-platforms.md [hunk 6]
```diff
@@ -45,12 +47,13 @@ but that the development team does not directly test or support.
...
+Note that Android SDK 20 is covered by testing Android SDK 19, 
+as the differences between the two platform versions are
+minimal.
 ### Best effort platforms tested by the community
 
 |Platform|Version       |
 |-------|---------------|
-|Android|Android SDK 30 |
 |Android|Android SDK 22 |
 |Android|Android SDK 20 |
```
Comment (dnfield) [inline src/docs/development/tools/sdk/release-notes/supported-platforms.md:58]: These can be removed

**Review by sfshaza2 — APPROVED** [review]
LGTM

**Pull request merged**